REQUIRE

```diff
@@ -1,11 +1,11 @@
-julia 0.6
-Missings 0.2.3
-CategoricalArrays 0.3.6
-StatsBase 0.11.0
-SortingAlgorithms
-Reexport
-WeakRefStrings 0.4.0
-DataStreams 0.3.0
-CodecZlib 0.4
-TranscodingStreams
-Compat 0.59.0
+julia 0.6

```

Comment:
I'm not sure what you changed here but since it only affects whitespace, please leave this file alone.

Reply:
Sorry, i thought I had fixed this. This is my first time doing a PR for anything, and my first time using Git, too.